adapters/appnexus.go

```diff
@@ -235,3 +237,42 @@ func NewAppNexusAdapter(config *HTTPAdapterConfig, externalURL string) *AppNexus
 		usersyncInfo: info,
 	}
 }
+
+func CollectImpData(oreq openrtb.BidRequest, req *pbs.PBSRequest, unit pbs.PBSAdUnit) string {
+	data := ""
+
+	//set 0 for web impression, and 1 for mobile impression
+	if oreq.Site != nil {
+		data += "plat:0"
+	} else {
+		data += "plat:1"
+	}
+
+	switch unit.MediaTypes[0] {
+	case pbs.MEDIA_TYPE_BANNER:
+		data += ";fmt:0"
+	case pbs.MEDIA_TYPE_VIDEO:
+		data += ";fmt:1"
+	default: //todo change to/add case for pbs.MEDIA_TYPE_NATIVE when it's implemented
```

Comment:
if we are not setting a default mediatType this could hit more than we intend. Just check to make sure we send banner in the default case.

Reply:
Hey Matt,

To clarify - do you mean use banner in default case for the logging statement (ie change the text to ";fmt:0" or in the actual PBSAdUnit object?

I've been doing some additional review of the code in the appnexus adapter and the openrtb_util.go file in how it handles MediaTypes.  It seems that there is a mechanism to 'filter' through the specified MediaTypes within the pbsadunit object setup within the bidder and only use the types that were noted in the adapter as acceptable.

While this logic is acting as a safe-guard, it isn't the same as setting a default inside the pbsadunit object and could allow someone to specify a value that's not currently supported.  If they did, then it would still be logged under the current default value in the logic I created.

Reply:
OK if this is coming from the adapter supported type it should be fine. Thanks for clarifying.